### DISCUSSION TITLE
Did someone fork why-cant-composer-load-repositories-recursively.md into a solution?

### DISCUSSION THREAD

**Discussion by siims-biz** [composer/composer#11371] [General]
There are 6.5K forks around.
Did someone fork why-cant-composer-load-repositories-recursively.md into a solution?
https://getcomposer.org/doc/faqs/why-cant-composer-load-repositories-recursively.md

Recently I experienced the problem and can live with the impact.
Just out of curiosity, is anyone aware of a fork which attempts to solve the issue?

**Comment by stof** [comment]
I don't know of any fork implementing this. But this would require first finding a (deterministic) solution to the problem of conflicting repository definitions (and it is quite incompatible with the concept of repository priority).
